Droidcon-Boston/app/src/main/res/layout/agenda_detail_fragment.xml

```diff
@@ -133,6 +133,39 @@
                 android:textColor="@android:color/black"
 
                 tools:text="@tools:sample/lorem/random" />
+            <RelativeLayout
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:orientation="horizontal"
+                android:layout_below="@id/tv_agenda_detail_description"
```

Comment:
is this impacting the placement at all?  Since it's in a linearlayout, i think we might be able to get rid of this line